partner_contact_nutrition_diet/data/nutrition_diet.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+ <odoo>
```

Comment:
Put noupdate attribute on the odoo element. User might want to change the descriptions in your records.